pyspecProcScripts/correlation_alignment.py

```diff
@@ -170,15 +178,21 @@ def correl_align(
                     ax=ax_list[1],
                 )
                 ax_list[1].set_title("correlation function (t), \nafter apod")
+        # we want Δnu centered at 0 and don't care about the previous window
+        # selection
         correl.ft_new_startpoint(direct, "freq")
         correl.ft_new_startpoint(direct, "time")
+        # Part of convolution is multiplying in t domain and then zero-filling
+        # before re-FT
         correl.ft(direct, shift=True, pad=2**14)
-        for k, v in signal_pathway.items():
-            correl.ft(["Delta%s" % k.capitalize()])
+        # {{{ Isolate the correlation function for the signal pathway
```

Comment:
I think this is what we are doing?

Reply:
I don't like the choice of loop variables here -- I would call these `ph_name` and `ph_val` or something like that.